components/memberstack/actions/create-member/create-member.mjs

```diff
@@ -0,0 +1,73 @@
+import memberstack from "../../memberstack.app.mjs";
+import pickBy from "lodash.pickby";
+
+export default {
+  key: "memberstack-create-member",
+  name: "Create Member",
+  description: "Creates a member connected to a free plan. [See the docs](https://memberstack.notion.site/Admin-API-5b9233507d734091bd6ed604fb893bb8)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    memberstack,
+    email: {
+      type: "string",
+      label: "Email",
+      description: "Email address of the new member",
+    },
+    password: {
+      type: "string",
+      label: "Password",
+      description: "Password for the new member. Must be at least 8 characters long.",
+    },
+    plans: {
+      type: "string[]",
+      label: "Plans",
+      description: "An array of planId objects. Example: `{{ [{\"planId\": \"pln_abc\"}] }}`",
+      optional: true,
+    },
+    customFields: {
+      propDefinition: [
+        memberstack,
+        "customFields",
+      ],
+    },
+    metaData: {
+      propDefinition: [
+        memberstack,
+        "metaData",
+      ],
+    },
+    loginRedirect: {
+      type: "string",
+      label: "Login Redirect",
+      description: "Optional login redirect URL",
+      optional: true,
+    },
+  },
+  async run({ $ }) {
+    const plans = this.plans?.length > 0
+      ? this.plans.map((plan) => (typeof plan === "object")
+        ? plan
+        : JSON.parse(plan))
+      : undefined;
+
+    let response;
+    try {
+      const { data } = await this.memberstack.createMember(pickBy({
+        email: this.email,
+        password: this.password,
+        plans,
+        customFields: this.customFields,
+        metaData: this.metaData,
+        loginRedirect: this.loginRedirect,
+      }));
+      response = data;
+    } catch (e) {
+      throw new Error(e.message);
+    }
+
+    response && $.export("$summary", `Successfully created member with ID ${response?.id}.`);
```

Comment:
Could you please use regular `if` to execute this statement? In my view, using `&&` makes readability a little bit worse. Especially for non-senior devs or something like that.